.gitignore

```diff
@@ -7,12 +7,16 @@ build
 .classpath
 .DS_Store
 bin/
+ca-certificates/*
```

Comment:
Is this necessary in the current PR?

Reply:
In the past, I needed to place the EMC certs in this directory to get anything to build in Docker in our lab. I just tested this today without the EMC certs and it works just fine. I can remove everything related to ca-certificates (in .gitignore and Dockerfile). However, we may want to keep it to make it easy to add CA certificates if they should become necessary in the future.